bashrc

```diff
@@ -0,0 +1,8 @@
+# Note that this file is overridden by newer versions of `nextstrain shell` and
```

Comment:
_(non-blocking)_

My text editor (VSCode) doesn't understand that this is a bash script. It tries to be "smart" and thinks it's Markdown instead, likely based on the `#` in the first few lines.

A proper shebang fixes that, though it might not be necessary/appropriate for this kind of file?

Reply:
Ah, yeah I didn't include a shebang because rc files are meant to be `source`d not directly invoked. For that same reason, they're not marked executable.

I generally think that file detection stuff should live outside of the files themselves, except in rare circumstances. That is, I'm not a huge fan of using modelines, though some people use them everywhere (and there is [a VS Code plugin](https://marketplace.visualstudio.com/items?itemName=chrislajoie.vscode-modelines) to respect them).